internal/exec/resolvable/resolvable.go

```diff
@@ -78,6 +84,24 @@ func (f *Field) Resolve(ctx context.Context, resolver reflect.Value, args interf
 	return wrapResolver(ctx, args)
 }
 
+func (f *Field) Validate(ctx context.Context, args interface{}) []error {
```

Comment:
Like for execution of requests which could fail across multiple fields and all be reported on the response, the intention here is to report _all_ validation errors from the request. Reporting only a single error can mean playing “whack a mole” resolving 1 error only to then have the next error reported.

Validation skips going deeper and validating child fields, since something like `hasRole` should apply to that entire subtree, but errors across different parts of the tree may be reported